chapters/001-getting-started/README.md

```diff
@@ -144,11 +144,11 @@ As the message indicates, that network is no longer available. `boudi` doesn't k
 
 ### Add our own IP address configuration
 
-Now that we've removed the default network docker created, let's get started creatig a network of our own! Let's add IP addresses to each of these containers using the `ip addr add` command. In this example, we want to use the `10.1.1.0/24` network for these containers. `10.0.0.0/8` is one of the networks identified in [RFC 1918](https://www.rfc-editor.org/rfc/rfc1918) that is exclusively used for _private_ networking. This means that any IP packet that reaches the internet with an IP address in this range will be dropped. This is helpful in our tutorial because if our system is misconfigured to route to the Internet, we don't want a false-positive for ping tests. Therefore on `pippin`, we use the command
+Now that we've removed the default network docker created, let's get started creating a network of our own! Let's add IP addresses to each of these containers using the `ip addr add` command. In this example, we want to use the `10.1.1.0/24` network for these containers. `10.0.0.0/8` is one of the networks identified in [RFC 1918](https://www.rfc-editor.org/rfc/rfc1918) that is exclusively used for _private_ networking. This means that any IP packet that reaches the internet with an IP address in this range will be dropped. This is helpful in our tutorial because if our system is misconfigured to route to the Internet, we don't want a false-positive for ping tests. Therefore on `pippin`, we use the command
 
-`ip addr add 10.1.1.3/24 dev eth0`
+`ip addr add 10.1.1.2/24 dev eth0`
 
-You'll want to repeat this process on `boudi`, but in this case the ip address is `10.1.1.2/24`.
+You'll want to repeat this process on `boudi`, but in this case the ip address is `10.1.1.3/24`.
```

Comment:
Pippin and Boudi are usually the "second seat" and "third seat" respectively elsewhere in the docs, so I thought this might be better for consistency.